components/brave_rewards/browser/database/publisher_info_database_unittest.cc

```diff
@@ -1293,6 +1293,26 @@ TEST_F(PublisherInfoDatabaseTest, Migrationv11tov12_ContributionInfo) {
   EXPECT_EQ(list.at(3)->publisher_key, "reddit.com");
 }
 
+TEST_F(PublisherInfoDatabaseTest, Migrationv12tov13) {
+  base::ScopedTempDir temp_dir;
+  base::FilePath db_file;
+  CreateMigrationDatabase(&temp_dir, &db_file, 12, 13);
+  EXPECT_TRUE(publisher_info_database_->Init());
```

Comment:
`EXPECT_TRUE` yields a fatal error, is this expected over `ASSERT_TRUE` which yields a non-fatal error

Reply:
it's vice-versa, ASSERT is fatal, EXPECT is non-fatal

Reply:
yeap I've got things back to front...